README.md

```diff
@@ -1,79 +1,88 @@
 # AnDOChecker
 
-Checks the validity of a directory with respect to the ANimal Data Organization (AnDO) specifications
+The Animal Data Organization (AnDO) is a set of specifications for the organization of a directory containing experimental data recorded in animals. It is very inspired by the BIDS specifications ( https://bids-specification.readthedocs.io ). The main difference lies in the fact that in experiments conducted with animals, it is very common to have different settings for different sessions, and more importantly for different animals (whereas research on human subjects tends to have exactly the same protocol used for all subjects, as looked after by the BIDS specifications).
 
-![version](https://img.shields.io/badge/version-1-informational)
+Futher information on Specs : https://int-nit.github.io/AnDOChecker/
 
 [![PyPI license](https://img.shields.io/pypi/l/ansicolortags.svg)](https://pypi.python.org/pypi/ansicolortags/)[![Generic badge](https://travis-ci.org/Slowblitz/BidsValidatorA.svg?branch=master)](https://shields.io/)[![made-with-python](https://img.shields.io/badge/Made%20with-Python-1f425f.svg)](https://www.python.org/)
 
-## Table of Contents
+## Installation
+### Dependencies
 
-- [AnDOChecker](#andochecker)
-  - [Table of Contents](#table-of-contents)
-  - [Use](#use)
-    - [Clone](#clone)
-    - [General usage](#general-usage)
-    - [Specific usage](#specific-usage)
-  - [Development](#development)
-    - [Testing usage](#testing-usage)
-  - [Details](#details)
+AnDOChecker requires:
 
-## Use
+- Python (>= 3.6)
+- Pip
 
-### Clone
-
-- Clone this repo to your local machine using `https://github.com/INT-NIT/AnDOChecker.git`
+### User installation
+```bash
+> Pip3 install AnDOchecker
```

Comment:
remove the capital at "Pip3"... and also, it should simply be "pip" (no need for the 3)

Reply:
Updated.